src/plugins/cosmos/components/modals/Send/hooks/useSendFees/useSendFees.tsx

```diff
@@ -0,0 +1,60 @@
+import { chainAdapters } from '@shapeshiftoss/types'
```

Comment:
Isn't that exactly the same as the existing `useSendFees`?

Reply:
yes - good catch